blocks/footer/footer.js

```diff
@@ -15,13 +29,72 @@ export default async function decorate(block) {
   if (resp.ok) {
     const html = await resp.text();
 
-    // decorate footer DOM
-    const footer = document.createElement('div');
-    footer.innerHTML = html;
+    // Create container for footer
+    const footerContainer = document.createElement('div');
+    footerContainer.classList.add('footer-container');
+    footerContainer.innerHTML = html;
+
+    const slogan = footerContainer.querySelector('div:nth-child(2) > p:nth-child(1)');
+    slogan.classList.add('footer-slogan');
+
+    footerContainer.querySelectorAll('ul')
+      .forEach((list, index) => {
+        list.classList.add(`footer-menu-list-${index}`);
```

Comment:
I find it easier to read in css when it's 1-based
```suggestion
        list.classList.add(`footer-menu-list-${index + 1}`);

```

Reply:
make sense. I like that too